store/httpstore.go

```diff
@@ -92,8 +93,38 @@ func (s HTTPStore) SetMeta(name string, blob json.RawMessage) error {
 	return err
 }
 
+func (s HTTPStore) SetMultiMeta(metas map[string]json.RawMessage) error {
```

Comment:
I specifically designed the stores to be dumb. They know about storing and retrieving content. Eventually I'll be switching out the `json.RawMessage` use to simply `[]byte`.

I thought for a long time about using Reader/Writer interfaces, but we always end up serializing json in memory so there isn't any gain to be had from those interfaces.